python/gps/algorithm/policy_opt/policy_opt_tf.py

```diff
@@ -137,6 +155,26 @@ def update(self, obs, tgt_mu, tgt_prc, tgt_wt):
         average_loss = 0
         np.random.shuffle(idx)
 
+        if self._hyperparams['fc_only_iterations'] > 0:
+            feed_dict = {self.obs_tensor: obs}
+            num_values = obs.shape[0]
+            conv_values = self.solver.get_last_conv_values(self.sess, feed_dict, num_values, self.batch_size)
+            for i in range(self._hyperparams['fc_only_iterations'] ):
+                start_idx = int(i * self.batch_size %
+                                (batches_per_epoch * self.batch_size))
+                idx_i = idx[start_idx:start_idx+self.batch_size]
+                feed_dict = {self.last_conv_vars: conv_values[idx_i],
+                             self.action_tensor: tgt_mu[idx_i],
+                             self.precision_tensor: tgt_prc[idx_i]}
+                train_loss = self.solver(feed_dict, self.sess, device_string=self.device_string, use_fc_solver=True)
+                average_loss += train_loss
+
+                if (i+1) % 500 == 0:
+                    LOGGER.info('tensorflow iteration %d, average loss %f',
+                                    i, average_loss / 500)
```

Comment:
Super nitpick, but could you change it to `i+1`, so that it matches the other output?